taipy/rest/config/rest_config.py

```diff
@@ -95,22 +144,20 @@ def ssl_key(self, value: Optional[str]):
     def ssl_context(self) -> Optional[Tuple[Optional[str], Optional[str]]]:
         return (self._ssl_cert, self._ssl_key) if self._use_https else None
```

Comment:
We must use the properties so the we replace the template by the environment variable value if ever.
```suggestion
        return (self.ssl_cert, self.ssl_key) if self._use_https else None
```